docs/config.json

```diff
@@ -341,6 +341,11 @@
               "title": "Resource Requests",
               "slug": "/access-controls/access-requests/resource-requests/",
               "forScopes": ["enterprise", "cloud"]
+            },
+            {
+              "title": "Access Requests in OSS Teleport",
+              "slug": "/access-controls/access-requests/oss-role-requests/",
+              "forScopes": ["oss", "enterprise", "cloud"]
```

Comment:
@zmb3 I'm guessing you'll want to adjust how this appears in the nav, just adding it since it didn't exist. Preview:

![image](https://user-images.githubusercontent.com/2349184/223170215-1ea82759-bbc9-408b-8a6c-908e81879b0b.png)

Reply:
Should I just do `forScopes: [ "oss" ]` here instead?

Reply:
Well... :pushes-glasses-up: doesn't this feature _technically_ also work on Enterprise? Like it's not OSS-only, as much as it's the only available option for OSS.

In any case, the `forScopes` field doesn't affect how it appears in the navbar, only what toggles are available on the page. So you could do that, and then the scopedBlock in the page would always render (which also makes it pointless).

Reply:
Good point. Will leave it.